packages/server/graphql/public/typeDefs/toggleSummaryEmail.graphql

```diff
@@ -0,0 +1,18 @@
+extend type Mutation {
+  """
+  Toggles the sendSummaryEmail value on the User object which determines whether summary emails are sent to the user
+  """
+  toggleSummaryEmail: ToggleSummaryEmailPayload!
```

Comment:
+1 probably an edge case, but did you consider adding the setting on team level? This way users could have this setting per team

Reply:
Yeah, there are plans to make these settings more advanced in the near future, but we just wanted to build something quick for now